src/services/coreBackService.ts

```diff
@@ -144,3 +144,26 @@ export const getProvenIdentityUserDetails = (
     ),
   );
 };
+
+export const getDcMawPoll = async (req: Request): Promise<boolean> => {
+  try {
+    // Added for the browser tests
+    if (config.ENABLE_PREVIEW) {
+      return true;
+    }
+
+    await axiosInstance.get(
+      config.API_DCMAW_POLL,
+      generateAxiosConfig(
+        `${config.API_BASE_URL}${config.API_DCMAW_POLL}`,
+        req,
+      ),
+    );
```

Comment:
Might be out of scope of this ticket but I understand this endpoint returns a `journeyType` response which will tell us where to take the user next - we'll need to use this to patch the 'continue' action on the spinner page so the user can continue, which might require storing it in the session 🤔

Reply:
Reading the ticket ACs it sounds like the continue action is meant to be in scope, could you take a look? @MikeCollingwood may be able to help as he originally worked on this endpoint